seed/php-sdk/imdb/src/ClientOptions.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Seed;
+
+use GuzzleHttp\Client;
+
+class ClientOptions
+{
+    /**
+     * The base URL for the API.
+     *
+     * @var string
+     */
+    public string $baseUrl = '';
+
+    /**
+     * The HTTP client used to make requests.
+     *
+     * @var Client
+     */
+    public Client $httpClient;
```

Comment:
I think we can have this be a `Psr\Http\Client\ClientInterface` which lets the user pass in any [PSR-18](https://www.php-fig.org/psr/psr-18/) compliant client! Guzzle's client implements this so I think that's the only change needed

Reply:
Great call, that's definitely what we should do 👍